src/app/shared/components/working-hours-form-wrapper/working-hours-form-wrapper.component.html

```diff
@@ -5,6 +5,6 @@
 </ng-container>
 <div fxLayout='row' fxLayoutAlign='center center'>
   <button class="btn-add" mat-button (click)="addWorkingHours()" [disabled]="workingHoursFormArray.invalid">
-    <mat-icon class="mat-icon-add">add</mat-icon>Додати ще одні Дні/Години
+    <mat-icon class="mat-icon-add">add</mat-icon>Додати ще один Розклад занять
```

Comment:
I think we should leave the old naming of the button, according to the design 
![image](https://user-images.githubusercontent.com/27493959/169820932-1ba7f7fc-be0d-47d3-9f00-1f08964d642b.png)

Reply:
done